modules/AOR_Reports/aor_utils.php

```diff
@@ -226,7 +226,7 @@ function getConditionsAsParameters($report, $override = array())
  * @param $date_time_period_list_selected
  * @return DateTime
  */
-function getPeriodDate($date_time_period_list_selected)
+function getPeriodDate($date_time_period_list_selected, $type)
```

Comment:
Hi @ojs87, thank you for your PR. Do we want to make `$type` mandatory? should we define it like the following?

```
function getPeriodDate($date_time_period_list_selected, $type = '')
```

Reply:
Hi @clemente-raposo , thanks for your feedback. I've updated the variable to make it non-mandatory as suggested!